cypress/e2e/CMS/$HomeAndLandingPage/index.js

```diff
@@ -460,4 +469,51 @@ And('every link in dynamic list starts with {string}', (espanolPath) => {
     cy.get("div[class='dynamic list'] a").invoke('attr', 'href').then((linkHref) => {
         expect(linkHref.startsWith(espanolPath)).to.be.true;
     })
-})
\ No newline at end of file
+})
+
+And('user clicks {string} button for adding hero image', (imageBtn) => {
+    cy.get(`div.paragraphs-dropbutton-wrapper`).find(`input[value="${imageBtn}"]`).click({ force: true });
+});
+
+And('user uploads test file {string} as {string} version', (fileName, mode) => {
+    if (mode == 'desktop') {
+        cy.fixture(fileName, { encoding: null }).as('fixture')
+        cy.get('input[type="file"]').first().selectFile('@fixture');
+        cy.wait(2000);
+        cy.get(`input[id*="edit-field-hero-banner-0-subform-field-${mode}-hero-0-alt"]`).type("Alt Text")
+    }
+    if (mode == 'tablet') {
+        cy.fixture(fileName, { encoding: null }).as('fixture')
+        cy.get('input[type="file"]').first().selectFile('@fixture');
+        cy.wait(2000);
+        cy.get(`input[id*="edit-field-hero-banner-0-subform-field-${mode}-hero-0-alt"]`).type("Alt Text")
+    }
+    if (mode == 'mobile') {
+        cy.fixture(fileName, { encoding: null }).as('fixture')
+        cy.get('input[type="file"]').first().selectFile('@fixture');
+        cy.wait(2000);
+        cy.get(`input[id*="edit-field-hero-banner-0-subform-field-${mode}-hero-0-alt"]`).type("Alt Text")
+    }
+});
+
+And('user enters {string} into {string} text field in home and landing page', (tagLineText, field) => {
+    cy.get(`input[id*="edit-field-hero-banner-0-subform-field-tagline-0-value"]`).type(tagLineText);
+});
+
+And('the banner image with the name {string} is displayed at {string} breakpoint', (imageName, mode) => {
+    if (mode == 'tablet') {
+        cy.get('div#nvcgSlHeroHeader source[media="(max-width: 1024px) and (min-width: 641px)"]').should('have.attr', 'srcset').and('include', imageName);
+    }
+    if (mode == 'desktop') {
+        cy.get('div#nvcgSlHeroHeader img').should('have.attr', 'src').and('include', imageName);
+    }
+});
+
+And('{string} text is displayed at {string} breakpoint', (fieldText, mode) => {
+    if (mode == 'tablet') {
+        cy.get('div#nvcgSlHeroHeader').find(`p:contains("${fieldText}")`).should('be.visible');
+    }
+    if (mode == 'desktop') {
+        cy.get('div#nvcgSlHeroHeader').find(`p:contains("${fieldText}")`).should('be.visible');
```

Comment:
so both if statements contains identical code expressions - you can remove both if and just leave cy.get line

Reply:
done